sys-power/upower/metadata.xml

```diff
@@ -4,6 +4,9 @@
 	<maintainer type="project">
 		<email>freedesktop-bugs@gentoo.org</email>
 	</maintainer>
+    <use>
+        <flag name="polkit">PolKit support in daemon through <pkg>sys-auth/polkit</pkg></flag>
+  	</use>
```

Comment:
Indendation is off, but also there is global IUSE policykit at your service.

Reply:
Done in https://github.com/gentoo/gentoo/commit/1e66f44078d9fbe93612bcdc8ade96abe8f3c506